SQF/dayz_code/compile/A_Plot_for_Life/fn_check_owner_friends.sqf

```diff
@@ -49,7 +49,7 @@ if (DZE_permanentPlot) then {
 	_owner = (_targetOwner == dayz_characterID);
 };
 
-if (_targetOwner in _friendlies) then {
+if (_player in _friendlies) then {
```

Comment:
You're right there is a mistake there.

This statement should be moved inside the ELSE right below the <code>_owner = </code> line:
```sqf
if (_targetOwner in _friendlies) then {_friendly = true};
```

And then below this line:
```sqf
_owner = (_playerUID == _targetOwner);
```

Add:
```sqf
if (_playerUID in _friendlies) then {_friendly = true};
```
Then it will correctly handle DZE_permanentPlot off and on.